dissect/target/plugins/apps/av/trendmicro.py

```diff
@@ -0,0 +1,122 @@
+from typing import Iterator
+
+from dissect import cstruct
+from dissect.util.ts import from_unix
+from flow.record.fieldtypes import path
+
+from dissect.target import Target
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+TrendMicroWFLogRecord = TargetRecordDescriptor(
+    "application/av/trendmicro/wflog",
```

Comment:
```suggestion
    "application/av/trendmicro/wf/log",
```
Same as for McAffee?